cluster-service/Makefile

```diff
@@ -72,13 +72,31 @@ deploy:
 	  --set azureOperatorsMI.imageRegistry.roleId="$${OP_IMAGE_REGISTRY_DRIVER_ROLE_ID}" \
 	  --set azureOperatorsMI.cloudNetworkConfig.roleName="${OP_CLOUD_NETWORK_CONFIG_ROLE_NAME}" \
 	  --set azureOperatorsMI.cloudNetworkConfig.roleId="$${OP_CLOUD_NETWORK_CONFIG_ROLE_ID}" \
+	  --set azureOperatorsMI.kms.roleName="${OP_KMS_ROLE_NAME}" \
+	  --set azureOperatorsMI.kms.roleId="$${OP_KMS_ROLE_ID}" \
+	  --set azureOperatorsMI.clusterApiAzure.roles[0].name="${OP_CLUSTER_API_AZURE_ROLE_NAME}" \
```

Comment:
I see it references roles, name and id.  The attribute names don't seem to match what's specified in the values.yaml file.
Is that correct and intended?

Reply:
no they are incorrect. 

Correted them now, please have a look.

Reply:
Although I see now we reference roleDefinitions I still see references to .name and .id instead of .roleName and .resourceId. Is that correct and intended? same applies to the references in the configmap yaml

Reply:
Yes that is intended. I kept it like that as this is an array of roleDefinitions.

Reply:
If it is intended, why is that some are referenced as .id, name ( azureOperatorsMI.kms.roleName)  and others as .roleId, .roleName ( azureOperatorsMI.clusterApiAzure.roleDefinitions[0].name) ?

Reply:
and how does that relate  to the attribute names in the values.yaml file ?

Reply:
`azureOperatorsMI.kms.roleName` this refer to the old schema of the configuration file which we need to keep until we deploy the new CS code. THe KMS one, I just shifted it with other similar flags just to keep them together.

Reply:
Understood 👍 . That makes sense to me